packages/compiler-sfc/src/cssVars.ts

```diff
@@ -13,7 +13,7 @@ import hash from 'hash-sum'
 
 export const CSS_VARS_HELPER = `useCssVars`
 export const cssVarRE =
-  /\bv-bind\(\s*(?:'([^']+)'|"([^"]+)"|([^'"][^)]*))\s*\)/g
+  /\bv-bind\(\s*(?:'([^']+)'|"([^"]+)"|([^'"][^;]*))\s*\)/g
```

Comment:
👍 


maybe we can squeeze  one more small improvement


allow whitespace between `v-bind` and the opening `(`
so that the the following css will be handled properly
```js
  .a {
    x: v-bind (dsa)
  }
```
currently it will be compiled to: 
```css
.a[data-v-f13b4d11] {
    x: v-bind (dsa)
}
```
 instead of 
```css
.a[data-v-f13b4d11] {
    x: var(--f13b4d11-dsa)
}
```
and that basically fails to setup the variable 

suggested change
```diff
-/\bv-bind\(\s*(?:'([^']+)'|"([^"]+)"|([^'"][^;]*))\s*\)/g
+/\bv-bind\s*\(\s*(?:'([^']+)'|"([^"]+)"|([^'"][^;]*))\s*\)/g
```